modules/core/core-api/src/main/java/com/enonic/xp/content/ContentVersionDateComparator.java

```diff
@@ -0,0 +1,40 @@
+package com.enonic.xp.content;
+
+import java.time.Instant;
+import java.util.Comparator;
+
+final class ContentVersionDateComparator
+    implements Comparator<ContentVersion>
+{
+    public static final ContentVersionDateComparator INSTANCE = new ContentVersionDateComparator();
+
+    private ContentVersionDateComparator()
+    {
+    }
+
+    @Override
+    public int compare( final ContentVersion thisVersion, final ContentVersion thatVersion )
+    {
+        Instant thisTime, thatTime;
```

Comment:
Codacy found an issue: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6220524)